sheet_to_triples/tests/test_rdf.py

```diff
@@ -12,30 +12,39 @@
 
 class TestRDF(unittest.TestCase):
 
+    @property
+    def resolver(self):
+        return rdf._new_graph().store.namespace
+
     def test_from_qname(self):
-        term = rdf.from_qname('owl:test')
+        term = rdf.from_qname('owl:test', self.resolver)
         self.assertIsInstance(term, rdflib.term.URIRef)
         self.assertEqual(str(term), 'http://www.w3.org/2002/07/owl#test')
 
     def test_from_qname_prefix_vm(self):
```

Comment:
Can probably junk this test now? It's no longer a discrete code path with special handling for `vm:` prefixes.